monash-human-power/MHP-website
# Migrate Yarn version from v1

## Description



Yarn version 1 has been [deprecated](https://yarnpkg.com/getting-started/migration#why-should-you-migrate) and will be unsupported in the future. I don't know when support will end, but to save myself (and everyone) the trouble, I've migrated yarn to the latest version.

## Screenshots



## Steps to Test



You might want to reinstall yarn to ensure you don't get the v1 version: <https://yarnpkg.com/getting-started/install>

Then just run `yarn install` and `yarn develop` as usual.
